plugins/services/src/js/components/forms/MultiContainerNetworkingFormSection.js

```diff
@@ -0,0 +1,452 @@
+import React from 'react';
+import {Tooltip} from 'reactjs-components';
+import mixin from 'reactjs-mixin';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import FieldHelp from '../../../../../../src/js/components/form/FieldHelp';
+import FieldInput from '../../../../../../src/js/components/form/FieldInput';
+import FieldLabel from '../../../../../../src/js/components/form/FieldLabel';
+import FieldError from '../../../../../../src/js/components/form/FieldError';
+import FieldSelect from '../../../../../../src/js/components/form/FieldSelect';
+import {findNestedPropertyInObject} from '../../../../../../src/js/utils/Util';
+import FormGroup from '../../../../../../src/js/components/form/FormGroup';
+import FormGroupContainer from '../../../../../../src/js/components/form/FormGroupContainer';
+import {FormReducer as network} from '../../reducers/serviceForm/MultiContainerNetwork';
+import HostUtil from '../../utils/HostUtil';
+import Icon from '../../../../../../src/js/components/Icon';
+import Networking from '../../../../../../src/js/constants/Networking';
+import VirtualNetworksStore from '../../../../../../src/js/stores/VirtualNetworksStore';
+
+const METHODS_TO_BIND = [
+  'onVirtualNetworksStoreSuccess'
+];
+
+class MultiContainerHealthChecksFormSection extends mixin(StoreMixin) {
```

Comment:
Let's try to create section parts we can re-use and then have one section instead of copying what looks to be almost the same as `NetworkingFormSection`